Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -518,27 +513,50 @@ - (void)loadVungleNativeAdForParameters:(id<MAAdapterResponseParameters>)paramet
     return clickableViews;
 }
 
-- (BannerSize)adSizeFromAdFormat:(MAAdFormat *)adFormat
+- (VungleAdSize *)adSizeFromAdFormat:(MAAdFormat *)adFormat
+                          parameters:(id<MAAdapterParameters>)parameters
 {
-    if ( adFormat == MAAdFormat.banner )
+    BOOL isAdaptiveBanner = [parameters.localExtraParameters al_boolForKey: @"adaptive_banner"];
+    NSNumber *customWidth = [parameters.localExtraParameters al_numberForKey: @"adaptive_banner_width"] ? : 0;
+    NSNumber *customHight = [parameters.localExtraParameters al_numberForKey: @"adaptive_banner_height"] ? : 0;
+
+    if (customWidth && customWidth) {
```

Comment:
Not sure if we need to add more check with isNotAdaptiveBanner here.

Reply:
That's good point. I will add the check.

Reply:
This is suppose to be if ( isAdaptive ).
there is no custome api for MAX.

Reply:
Currently, we are waiting for the answer from Product team on whether we need to support "custom size with **work-around** api" or not (and not supporting adaptive). 
Based on it, I will update this logic.